src/library/autodj/autodjprocessor.cpp

```diff
@@ -1665,6 +1669,20 @@ void AutoDJProcessor::playerRateChanged(DeckAttributes* pAttributes) {
     calculateTransition(fromDeck, getOtherDeck(fromDeck), false);
 }
 
+void AutoDJProcessor::playlistFirstTrackChanged() {
+    qDebug() << this << "playlistFirstTrackChanged";
```

Comment:
please wrap this in `if constexpr (sDebug) {` like the other debug logs.

Reply:
Done, thanks for your feedback.